### PR TITLE
[MID-80] add admin component conventions

### PR DESCRIPTION
**Pull request by IvanKoshan**
[Ticket](https://trello.com/c/REKec9L4/80). Added conventions for admin components and move icons to the assests

### CODE REVIEW

File: docs/CODESTYLE.md [hunk 1]
```diff
@@ -43,3 +43,42 @@ export const utility = () => {
   return ...
 };
 ```
+
+## Admin component folders/files conventions
+
+- Start folder names with capital letters. For ex: `Organiation` instead of `organizations`.
```
Comment (watchlar2000) [inline docs/CODESTYLE.md]: minor typo

File: docs/CODESTYLE.md [hunk 2]
```diff
@@ -43,3 +43,42 @@ export const utility = () => {
...
+- Filenames for files with feature components for specific model are `Create.js`, `Edit.js`, `List.js`, `Show.js`.
+- These files should contain components with name consisting of the **model name** and **feature name**, for ex: `FaqList` or `EventCreate`. Here is example of using them in `adminApp.js` in a resource.
```
Comment (watchlar2000) [inline docs/CODESTYLE.md]: adminApp.js -> AdminApp.js?

**Review by alex-popov-tech — APPROVED** [review]

File: docs/CODESTYLE.md (hunk 2 above)
Comment (IvanKoshan2004) [reply docs/CODESTYLE.md]: Renamed

File: docs/CODESTYLE.md (hunk 1 above)
Comment (IvanKoshan2004) [reply docs/CODESTYLE.md]: Fixed

**Review by watchlar2000 — APPROVED** [review]

File: src/app/_components/CheckBox/CheckBox.js [hunk 3]
```diff
@@ -2,9 +2,10 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import TickIcon from '@icons/tickIcon.svg';
+// import { TickIcon } from '../Icons';
```
Comment (yataras) [inline src/app/_components/CheckBox/CheckBox.js]: Please remove this line if you've changed the approach TickIcon is used. Also, - update `<TickIcon/>` usage below if it it just an .svg.

**Review by yataras — CHANGES_REQUESTED** [review]

File: src/app/_components/CheckBox/CheckBox.js (hunk 3 above)
Comment (IvanKoshan) [reply src/app/_components/CheckBox/CheckBox.js]: Removed the commented line. The usage was also updated. There must be classes added there to account for multiple checkbox states. Styles are updated and working as expected

**Review by Mykhailo-Pasternak — APPROVED** [review]
looks nice)

**Review by yataras — APPROVED** [review]

**Pull request merged**